src/applications/personalization/profile360/actions/misc.js

```diff
@@ -25,7 +25,7 @@ function inferAddressType(countryName, stateCode) {
   if (countryName !== 'United States') {
     addressType = 'INTERNATIONAL';
   } else if (MILITARY_STATES.has(stateCode)) {
-    addressType = 'MILITARY OVERSEAS';
+    addressType = 'OVERSEAS MILITARY';
   }
```

Comment:
After things get quiet I would like to get our Vet360 address logic more organized, getting these address-types stored in a constant.